src/main/java/com/be/two/c/apibetwoc/service/produto/ProdutoMapaService.java

```diff
@@ -25,9 +26,9 @@ public class ProdutoMapaService{
     public List<Produto> retornarProdutos(Double latitude,Double longitude,Double distancia,String produto,String metodoPagamento){
         List<Integer> estabelecimentos = estabelecimentoRepository.buscarPorLocalizacao(latitude,longitude,distancia);
         Specification<Produto> estabelecimentoSpecification = Specification.where(ProdutoSpecification.filtrarIds(estabelecimentos).and(ProdutoSpecification.name(produto).and(ProdutoSpecification.metodoPagamento(metodoPagamento))));
-        List<Produto> produtos = produtoRepository.findAll(estabelecimentoSpecification);
+        Page<Produto> produtos = produtoRepository.findAll(estabelecimentoSpecification,null);
         produtos.stream().forEach(produto1 -> produto1.setImagens(produto1.getImagens().stream().map(imagemService::formatterImagensURI).toList()));
-        return produtos;
+        return produtos.stream().toList();
```

Comment:
Porque esse toList() ?

Reply:
Perai, são coisas que eu to mexendo ainda kkkkk n tinha visto essa parte

Reply:
Vc botou esse toList pq ele devolvia uma page

Reply:
é pra devolver uma page msm

Reply:
Ixi fui eu kkkkkkkkkkk

Reply:
É para devolver um List